cmd/kueue/main.go

```diff
@@ -264,7 +264,7 @@ func main() {
 
 	if features.Enabled(features.VisibilityOnDemand) {
 		go func() {
-			if err := visibility.CreateAndStartVisibilityServer(ctx, queues); err != nil {
+			if err := visibility.CreateAndStartVisibilityServer(ctx, queues, *cfg.InternalCertManagement.Enable); err != nil {
```

Comment:
I wonder if `cfg.InternalCertManagement` could be nil, probably not based on the default setting, right?
https://github.com/kubernetes-sigs/kueue/blob/main/apis/config/v1beta1/defaults.go#L86-L87